cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -281,10 +281,10 @@ namespace Slice
             typedef map<string, string> StringMap;
             struct OpComment
             {
-                StringVec description;
-                StringMap params;
-                string returns;
-                StringMap exceptions;
+                vector<string> description;
+                map<string, vector<string>> params;
+                vector<string> returns;
+                map<string, vector<string>> exceptions;
```

Comment:
The param and exception descriptions were placed in a single line, better keep the lines from the original Slice file.